js/templates/userPage.html

```diff
@@ -246,7 +246,7 @@ <h1 class="page-userNameLarge floatLeft fancy-heading fontSize20 colorWhiteForce
         <div class="flexCol-8 custCol-border borderBottom0">
           <div class="rowItem userPageAboutSection js-userPageAboutSection padding2520">
             <% if(ob.ownPage) { %>
-              <a href="#settings/page">
+              <a href="#settings/page" class="userPageEditBtn">
```

Comment:
The tooltip is being created by some code in main.js, I'll add a check to only show it for external links.